packer/README.adoc

```diff
@@ -0,0 +1,74 @@
+= Packer build for the Streaming Date Workshop Virtual Machine
+
+This build creates a Fedora 26 virtual machine having:
+
+- `root` with password `packer`
+- `developer` user with password `password` (_sudoer_)
```

Comment:
Following OpenShift's lead of using `developer`/`developer` as usr/pwd for entering the console, I've used that combination for the management access to data grid (for visualizer). Maybe we want to use the same combo here too?